operators/polyhedral_splines.py

```diff
@@ -78,19 +84,46 @@ def __init_patch_obj__(self, context):
         bm.faces.ensure_lookup_table()
 
         # Iterate through each vert of the mesh
+        runningSum = 0.000
+        centerOfMass = numpy.zeros(3)
+        momentOfInertia = numpy.zeros((3,3))
         for v in bm.verts:
             # Iterate throgh different type of patch constructors
             for pc in self.vert_based_patch_constructors:
                 if pc.is_same_type(v):
                     start = time.process_time()
-
                     bspline_patches = pc.get_patch(v)
                     patch_names = PatchOperator.generate_multiple_patch_obj(bspline_patches)
                     nb_verts = pc.get_neighbor_verts(v)
                     PatchTracker.register_multiple_patches(v, nb_verts, patch_names)
-
+                    bezier_patches = pc.get_bezier_patch(v)
```

Comment:
Please move the code related to moment to another file (e.g. moment.py). Amd leave as few function calls in this file (main file) as possible. Thanks!